homeassistant/components/frontend/services.yaml

```diff
@@ -1,14 +1,25 @@
 # Describes the format for available frontend services
 
 set_theme:
+  name: Set theme
   description: Set a theme unless the client selected per-device theme.
   fields:
     name:
+      name: Name
       description: Name of a predefined theme, 'default' or 'none'.
+      required: true
       example: "default"
+      selector:
+        text:
     mode:
+      name: Mode
       description: The mode the theme is for, either 'dark' or 'light' (default).
```

Comment:
```suggestion
      name: Mode
      description: The mode the theme is for, either 'dark' or 'light'.
      default: "light"
```

Reply:
Is there a guideline whether descriptions should end with a dot or not? In this PR one service field does, the other not.

Reply:
In general, specifying the default, is better than describing it.

Reply:
I wasn't talking about the "default", but the "dot" 😉.

Reply:
Aah yes, descriptions should end with a full stop `.`. This has changed over time as well, as we originally didn't have a name (and thus description was part of the list shown).